README.md

```diff
@@ -104,6 +104,9 @@ npm run lint
 npm run lint:fix
 ```
 
+### Storybook
+To start storybook locally run `npm run storybook`.
```

Comment:
Would it be useful to just wrap this into `dev-start.sh`?

Reply:
Added to `dev-start.sh`.